### ISSUE TITLE
slcli incompatibility with latest Click library (7.0)

### ISSUE THREAD

**Issue by kramimus** [softlayer/softlayer-python#1040]
### Expected Behavior
slcli commands should return error codes of zero if there are no real issues

### Actual Behavior
slcli command runs but then always returns a non-zero exit code with an exception, for example:
```
$ slcli vm list
:..........:...................:................:................:............:........:
:    id    :      hostname     :   primary_ip   :   backend_ip   : datacenter : action :
:..........:...................:................:................:............:........:
<VALID VM LIST OUTPUT>
:..........:...................:................:................:............:........:
An unexpected error has occured:
Traceback (most recent call last):
  File "/tmp/venv/local/lib/python2.7/site-packages/SoftLayer/CLI/core.py", line 182, in main
    cli.main(**kwargs)
  File "/tmp/venv/local/lib/python2.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/tmp/venv/local/lib/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/tmp/venv/local/lib/python2.7/site-packages/click/core.py", line 1102, in _process_result
    **ctx.params)
  File "/tmp/venv/local/lib/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/tmp/venv/local/lib/python2.7/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/tmp/venv/local/lib/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
TypeError: output_diagnostics() got multiple values for keyword argument 'verbose'

Feel free to report this error as it is likely a bug:
    https://github.com/softlayer/softlayer-python/issues
The following snippet should be able to reproduce the error
```

### Environment Information
Operating System: Ubuntu 16.04.5
softlayer-python version (`slcli --version`): slcli (SoftLayer Command-line), version 5.5.2

The version of Click causing the issue is 7.0. One solution appears to be pinning Click to version 6.7 or <=6.7

**Comment by allmightyspiff** [comment]
As part of #1042 I'll look into this library as well.

**Comment by allmightyspiff** [comment]
#1055 updates slcli to support click v7, and will be included in v5.6.0
